docs/proposals/ListView.md

```diff
@@ -0,0 +1,154 @@
+# Component Name
+
+The `ListPage` component will provide the structure and parts needed to display
```

Comment:
`ListPage` or `ListView`? the title of the PR is `ListView`

Reply:
I'm not sure. Do you have any thoughts on that? I started with View then realized it sort of is a whole page…?

Reply:
Moved things to ListPage after talking.

Reply:
^ This proposal file needs a rename, then.